src/game/gui/text_render.h

```diff
@@ -77,16 +78,46 @@ typedef struct {
     uint8_t max_lines;
 } text_settings;
 
+typedef struct letter {
+    int x;
+    int y;
+    int offset;
+    int limit;
+    surface *sur;
```

Comment:
```suggestion
    surface const *sur;
```
Since this is not an owned surface, and we do not plan to modify the surface via this pointer.